### PR TITLE
(WIP) ChestholeTilesets: Back to no ASCII

### PR DESCRIPTION
**Pull request by Chezzo**
This completes the two or three terrain (✔) and the 180 items ( ) I have left to get my tile sets to no ASCII on vanilla.

It also fixes things like the stray pixel line on textbooks, and general cleanup of some tiles.

### CODE REVIEW

File: gfx/ChestHole16Tileset/tile_config.json [hunk 1]
```diff
@@ -18869,7 +18869,7 @@
           "rotates": false
         },
         {
-          "id": "45_casing",
+          "id": ["45_casing", "357sig_casing"],
           "fg": 58,
```
Comment (BorkBorkGoesTheCode) [inline gfx/ChestHole16Tileset/tile_config.json]: Why not base it off bottleneck 7.62x25?

Comment (BorkBorkGoesTheCode) [reply gfx/ChestHole16Tileset/tile_config.json]: Oh, so the 7.62x25 is using the 120mm tank shell canister sprite.

Comment (BorkBorkGoesTheCode) [reply gfx/ChestHole16Tileset/tile_config.json]: Here's a lineup: 9mm 7 62x25mm 357 sig 10mm 45sw 45gap 50ae
9mm_7 62mm_357sig_10mm_45sw_45gap_50ae_002

**Comment by ZhilkinSerg** [comment]
Is it ready to be merged yet?

**Comment by chriseich21** [comment]
Would also be interested in seeing it merged in, current version of CDDA is still missing some pretty common ones like wardrobes, skeletal juggernaut, metal filing cabinet. Great work on tiles though btw

**Comment by kevingranade** [comment]
Needs a rebase or re-doing, feel free to reopen if it should stick around.

**Comment by Throwaway-name** [comment]
@Chezzo

**Pull request closed without merging**